drivers/platform/aducm3029/iio_aducm3029.h

```diff
@@ -113,13 +106,6 @@ enum iio_gpio_attributes {
 	.show = get_gpio_attr,\
 	.store = set_gpio_attr\
 }
-static struct iio_attribute gpio_attributes[] = {
-	IIO_GPIO_ATTR("en", GPIO_ENABLE),
-	IIO_GPIO_ATTR("value", GPIO_VALUE),
-	IIO_GPIO_ATTR("is_output", GPIO_DIRECTION_OUTPUT),
-	IIO_GPIO_ATTR("number", GPIO_NUMBER),
-	END_ATTRIBUTES_ARRAY,
-};
 
 #define ADCUM3029_ADC_CH(idx)  {\
```

Comment:
You can move this too to .c file to be all in one place. These macros won't be used elsewhere